assets/src/dashboard/app/views/shared/typeaheadSearch.js

```diff
@@ -32,12 +32,18 @@ export default function TypeaheadSearch({
 }) {
   const typeaheadMenuOptions = useMemo(() => {
     // todo add different option sets, value and label won't always be the same
-    return stories.map((story) => {
-      return {
-        label: story.title,
-        value: story.title,
-      };
-    });
+    return stories.reduce((acc, story) => {
+      if (!story.title) {
```

Comment:
If we want to trim whitespace only titles we should also do `!story.title || story.title.trim() === ''`

Reply:
great point, i'll update in the am